src/mbgl/tile/vector_tile.cpp

```diff
@@ -15,9 +15,17 @@ class VectorTileLayer;
 
 using packed_iter_type = protozero::iterator_range<protozero::pbf_reader::const_uint32_iterator>;
 
+struct VectorTileLayerData {
```

Comment:
Yes, that's correct. One minor note is that `shared_ptr<const std::string>` predated the addition of `VectorTileLayerData` and comes from the definition of [`Response`](https://github.com/mapbox/mapbox-gl-native/blob/b404457a74d7a63103285712552a8564823db6d9/include/mbgl/storage/response.hpp#L31). I believe this was originally done to avoid string copies. Might be better to use move semantics now.